src/bosh-monitor/lib/bosh/monitor/resurrection_manager.rb

```diff
@@ -0,0 +1,111 @@
+module Bosh::Monitor
+  class ResurrectionManager
+    def initialize()
+      @parsed_rules = []
+      @logger = Bhm.logger
+      @active_id = "-1"
+    end
+
+    def resurrection_enabled?(deployment_name, instance_group)
+      enabled = true
+      @parsed_rules.each do |parsed_rule|
+        enabled = enabled && parsed_rule.enabled? if parsed_rule.applies?(deployment_name, instance_group)
+      end
+
+      return enabled
+    end
+
+    def update_rules(resurrection_config)
+      new_parsed_rules = []
+      if !resurrection_config.nil? && !resurrection_config.empty?
+        content = resurrection_config.first['content']
+        content_hash = YAML.load(content)
+        id = resurrection_config.first['id']
+
+        if @active_id != id
+          @logger.info("Resurrection config update starting...")
+
+          resurrection_rule_hashes = content_hash.fetch('resurrection', [])
```

Comment:
let's rename `resurrection` to `rules`. i've updated bosh-notes as well: https://github.com/cloudfoundry/bosh-notes/blob/master/proposals/resurrection-config.md.